src/main/java/graph/SequenceNode.java

```diff
@@ -1,31 +1,136 @@
-package graph; /**
- * Created by Lex on 25-4-17.
- */
-
+package graph;
 import java.util.ArrayList;
 
 /**
  * Class Node2, which represents sequences of DNA. A sequence is a part of a genome.
  * The sequence is a String consisting of A, C T and G.
  */
-public class SequenceNode extends Node {
-
+public class SequenceNode extends AbstractNode {
     private int id;
+    private String seq;
+    private ArrayList<Integer> adjacencyList;
+    private ArrayList<Integer> parents;
     private ArrayList<Integer> ids;
+    private int index;
+    private int layer;
+    private float baryCenterValue;
+    private int xCoordinate;
+    private int yCoordinate;
 
-    public SequenceNode(int id) {
+
+    public SequenceNode(int id, String seq){
         this.id = id;
+        this.seq = seq;
+        this.adjacencyList = new ArrayList<Integer>();
+        this.parents = new ArrayList<Integer>();
```

Comment:
Maybe this is stiil too much information for in a node. But that is a big change we can do later. Looks good anyway.